rover-apps/science/src/AdafruitSTEMMAModule.cpp

```diff
@@ -0,0 +1,13 @@
+#include "AdafruitSTEMMAModule.h"
+#include "Module.h"
+#include "mbed.h"
+#include "AdafruitSTEMMA.h"
+
+AdafruitSTEMMAModule::AdafruitSTEMMAModule() {
+    moistureSensorModule = new AdafruitSTEMMA(TEMP_MOIST_I2C_SDA, TEMP_MOIST_I2C_SCL);
+    bool resetStatus = moistureSensor -> reset();
+}
+
+void AdafruitSTEMMAModule::periodic_100ms(void) {
+    bool updateStatus = moistureSensor -> update();
```

Comment:
Originally I was thinking of keeping all the CAN signal updates in a separate CAN module, but it might make more sense to but the moisture sensor related CAN signal updates in this module instead. Can you leave a comment here reminding to add the CAN stuff once the CAN driver module is done?

We'll need to figure out if we even want a CAN app module

Reply:
If `update()` returns false then there was an error - you can just log it with printf for now, we'll switch to CAN later